sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ReceiveMessageStreamSyncSample.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.core.util.IterableStream;
+
+import java.time.Duration;
+
+/**
+ * Sample demonstrates how to receive a stream of {@link ServiceBusReceivedMessage} from an Azure Service Bus Queue
```

Comment:
It's not a stream. You received a batch of, at most, 5 messages from the service bus queue. The data structure we return could be streamed.